neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -316,7 +315,23 @@ private void OnVersionMessageReceived(VersionPayload payload)
 
         private void OnDisconnectMessageReceived(DisconnectPayload payload)
         {
-            system.LocalNode.Tell(payload, Context.Parent); // As the priority of message is lower than Tcp.Close
+            switch (payload.Reason)
+            {
+                case DisconnectReason.MaxConnectionReached:
+                case DisconnectReason.MaxConnectionPerAddressReached:
+                    try
+                    {
+                        var addressList = payload.Data
+                            .AsSerializableArray<NetworkAddressWithTime>(AddrPayload.MaxCountToSend)
+                            .Select(p => p.EndPoint)
+                            .Where(p => p.Port > 0);
+                        system.LocalNode.Tell(new Peer.Peers { EndPoints = addressList });
+                    }
+                    catch { }
+                    break;
+                default: break;
+            }
+            Context.Stop(Self);
```

Comment:
I need to have a test here in my private net, when I wrote the code here ago, some error have occurred, the `LocalNode` could not received the peers

Reply:
Test failed, should change ` Context.Stop(Self)`  to ` Context.Stop(Context.Parent)` in `ProtocolHandler.OnDisconnectMessageReceived`, otherwise the connection will not close.

Reply:
Fixed, work well in my local network(4 consensus nodes, 3 clients), and waiting for @cloud8little 

![微信图片_20191126180529](https://user-images.githubusercontent.com/43396869/69619974-8367ec00-1077-11ea-8211-d279e00eb6d8.png)